### DISCUSSION TITLE
spaCy to find family links between people

### DISCUSSION THREAD

**Discussion by lcristancho** [explosion/spaCy#5448] [Help: Best practices]
Currently I’m using spaCy to analyze a series of genealogies of Bogotá. The purpose of the investigation is to track last names of Encomenderos (the encomienda was an institution implemented by the Spanish conquistadors during colonization in America to take advantage of indigenous workforce) to the XVIII-XIX higher education institutions in order to prove the existence (or nonexistence) of social mobility. 

We would like to use spaCy to identify the family relationships that exist between the individuals of the genealogies. I decided to write you this Email to ask you for advice on how to execute this task. So far, we have been able to extract all the names that are contained in the books. However, the Genealogies we are working are written old books and do not have a systematic way of describing family relationships (as opposed to, for example, notarial records). Because of this, we haven’t been able to figure out a way to extract these links (in most literature, authors handle historical records that describe family relationships in a more systematical way).

Any help could be very useful.

Thank you!

**Comment by svlandeg** [comment]
Hi @lcristancho, happy to hear you're finding spaCy useful in your project. In general I'm afraid we don't really have the bandwith to advice in detail about specific projects, and we try to keep this Issue Tracker focused on bug reports and feature requests. I would generally advice to ask these kind of questions on StackOverflow, but it would be good to have a minimal example of working code that shows some examples of the texts you're processing, the results you're already obtaining (I'm guessing NER to extract PERSON entities?), and what is currently missing and what the desired outcome on your example texts would be.

If you decide to write that StackOverflow post with more background and some example code & data, feel free to link it here and I'll try to have a look when I have some time. In the meantime, I'll close this issue here as there's no immediate action point on our side (you'll still be able to reply).

[EDIT:] moved from the Issue Tracker to the Discussion forum.
